responsibleai/responsibleai/rai_insights/rai_insights.py

```diff
@@ -714,8 +714,11 @@ def _load_model(inst, path):
             inst.__dict__[_MODEL] = serializer.load(top_dir)
         else:
             inst.__dict__['_' + _SERIALIZER] = None
-            with open(top_dir / _MODEL_PKL, 'rb') as file:
-                inst.__dict__[_MODEL] = pickle.load(file)
+            try:
```

Comment:
not necessary but might be interesting to add a test for this scenario

Reply:
Added the test case.